benchmark/testsystems/alanine_dipeptide.py

```diff
@@ -41,30 +41,30 @@ def load_solvated_alanine(constrained=True):
 alanine_constrained = EquilibriumSimulator(platform=configure_platform("Reference"),
                                            topology=top, system=sys, positions=pos,
                                            temperature=temperature,
-                                           ghmc_timestep=1.0 * unit.femtosecond,
+                                           xcghmc_timestep=1.0 * unit.femtosecond,
```

Comment:
Changed variable name, propose to address the second comment when this is incorporated into `openmmtools` in the interest of time.